packages/@react-spectrum/icon/docs/Icon.mdx

```diff
@@ -25,12 +25,32 @@ category: Misc
 ---
 
 # Icon
-<p>Docs coming soon</p>
+
+<p>{docs.exports.Icon.description}</p>
 
 ## Available Icons
 
 ### Workflow
-<HeaderTable name={workflowIconPackageData.name} version={workflowIconPackageData.version} />
+
+<HeaderInfo
+  packageData={workflowIconPackageData}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/icons/'}
+  ]} />
+
+<br />
+
+Spectrum workflow icons are graphical metaphors or symbols that users interact with to navigate and manipulate objects.
+They all appear at the same size within each platform, 18 px for desktop and 22 px for mobile. See [ActionButton](/react-spectrum/ActionButton.html#content)
```

Comment:
Don't know if we need to say anything, but actually, just noticed, this isn't strictly true.
That's the size that they render at, but it's not the defined size, i believe they are defined at 18x18 and everything else is just scaling that, at least, until spectrum-icons comes out with their 24x24 size or something